test/e2e/specs/site-editor/push-to-global-styles.spec.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * WordPress dependencies
+ */
+const {
+	test,
+	expect,
+	Editor,
+} = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	editor: async ( { page }, use ) => {
+		await use( new Editor( { page, hasIframe: true } ) );
+	},
+} );
+
+test.describe( 'Push to Global Styles button', () => {
```

Comment:
What do you think about a test for undo?

Reply:
I didn't bother because planning (in January) on removing the custom undo functionality in favour of native undo/redo.